modules/atomic-widgets/props-resolver/transformers/styles/background-image-overlay-transformer.php

```diff
@@ -50,4 +47,20 @@ private function get_image_url_by_id( $id ): string {
 
 		return $image_url;
 	}
+
+	private function get_position_and_size_style( array $value ): string {
+		if ( ! isset( $value['size'] ) && ! isset( $value['position'] ) ) {
+			return '';
+		}
+
+		if ( ! isset( $value['size'] ) ) {
+			return ' ' . $value['position'];
+		}
+
+		$default_position = '0% 0%';
+
+		$position = $value['position'] ?? $default_position;
+
+		return ' ' . $position . ' / ' . $value['size'];
```

Comment:
```suggestion
		return  $position . ' / ' . $value['size'];
```

I would add it in the transformer itself

Reply:
Done and solved with the use an array and implode method.